cmd/power_user.go

```diff
@@ -1,170 +0,0 @@
-package cmd
-
-import (
-	"fmt"
-	"sync"
-
-	"github.com/anchore/syft/syft/artifact"
-
-	"github.com/anchore/syft/syft/sbom"
-
-	"github.com/anchore/stereoscope"
-	"github.com/anchore/syft/internal"
-	"github.com/anchore/syft/internal/bus"
-	"github.com/anchore/syft/internal/log"
-	"github.com/anchore/syft/internal/presenter/poweruser"
-	"github.com/anchore/syft/internal/ui"
-	"github.com/anchore/syft/syft/event"
-	"github.com/anchore/syft/syft/source"
-	"github.com/pkg/profile"
-	"github.com/spf13/cobra"
-	"github.com/wagoodman/go-partybus"
-)
-
-const powerUserExample = `  {{.appName}} {{.command}} <image>
-
-  Only image sources are supported (e.g. docker: , docker-archive: , oci: , etc.), the directory source (dir:) is not supported.
-
-  All behavior is controlled via application configuration and environment variables (see https://github.com/anchore/syft#configuration)
-`
-
-var powerUserOpts = struct {
-	configPath string
-}{}
-
-var powerUserCmd = &cobra.Command{
```

Comment:
how do we feel about adding a deprecation notice to the command and removing in 1.0?

Reply:
Happy to do that - was very excited about deleting a chunk of code